wiki/osu!supporter/es.md

```diff
@@ -1,24 +1,22 @@
-![Etiqueta de osu!supporter](Supporter tag.png "Etiqueta de osu!supporter")
+# osu!supporter (Español) 
 
-Introducción
-============
+![Etiqueta de osu!supporter](/wiki/shared/osu!supporter.png "Etiqueta de osu!supporter")
 
-<img src="Osu-support.png" title="Enlace de osu!supporter en la página principal." alt="Enlace de osu!supporter en la página principal." width="200" />
+![Enlace a osu!supporter en la página principal](Osu-support.png "Enlace a osu!supporter en la página principal")
 
-Para más información, consulte el enlace: <http://osu.ppy.sh/p/support>
+Consulte el enlace: <http://osu.ppy.sh/p/support> (En inglés)
 
 En resumen: las donaciones son usadas en su totalidad para mantener osu!
 
-Ventajas del osu!supporter
-==========================
+## Ventajas del osu!supporter
 
--   osu!direct (Descargas directas adentro del juego)
+-   osu!direct (Descargas directas dentro del juego)
 -   Acceso al multijugador usando osu!cuttingedge.
 -   [Chat](Chat "wikilink"): Nombre en <span style="color:#FFC700">Amarillo brillante</span> (para los demás, te seguirás viendo con el nombre blanco, como siempre).
```

Comment:
wikipedia syntax won't be supported

and remove `<span/>`

Reply:
I cant find one that actually works.